src/rate/src/Rate.tsx

```diff
@@ -131,9 +136,11 @@ export default defineComponent({
             onClick={() => this.handleClick(index)}
             onMouseenter={() => this.handleMouseEnter(index)}
           >
-            <NBaseIcon clsPrefix={mergedClsPrefix}>
-              {{ default: () => StarIcon }}
-            </NBaseIcon>
+            {
+              defaultSlot?.() ?? <NBaseIcon clsPrefix={mergedClsPrefix} size={this.size}>
```

Comment:
The size setting pattern doesn't follow naive ui's style. Use theme variable instead. You can have a look at avatar to know how to set it.

![image](https://user-images.githubusercontent.com/18677354/121780816-95b36780-cbd4-11eb-82f1-caba50542e31.png)

![image](https://user-images.githubusercontent.com/18677354/121780824-a368ed00-cbd4-11eb-9825-40d9ed4c475c.png)